### PR TITLE
Fix telegram username form

### PR DESCRIPTION
**Pull request by therizhao**
## Describe your changes

- Temp fix for iframe styling
  - No simple way to style Telegram iframe
  - Saw an option here https://stackoverflow.com/questions/56347902/telegram-authorization-without-default-button
    - But not sure how to integrate with our UI
- Temp fix for onboarding
  - Auto populate ProfileEditForm by setting isEdit to true
  - Auto populate Username with Telegram username
  - This code can be deleted when @CrownKira is done with onboarding flow

## Issue ticket number and link

#60 

## Screenshots (if appropriate):

image
image

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

**Review by Artemis-Hunt — COMMENTED** [review]
LGTM! (Besides the typo below)

File: frontend/src/sections/auth/login/LoginForm.tsx [hunk 1]
```diff
@@ -1,29 +1,49 @@
...
 import { BE_API } from 'src/utils/api';
 // @ts-ignore (react-telegram-login is missing a @types file)
 import TelegramLoginButton from 'react-telegram-login';
+import { useEffect } from 'react';
 
 // ----------------------------------------------------------------------
 
 export default function LoginForm() {
+  useEffect(() => {
+    document.querySelector('ifrmae');
```
Comment (Artemis-Hunt) [inline frontend/src/sections/auth/login/LoginForm.tsx:15]: Typo iframe

File: frontend/src/sections/auth/login/LoginForm.tsx [hunk 2]
```diff
@@ -1,29 +1,49 @@
...
+  }, []);
```
Comment (chownces) [inline frontend/src/sections/auth/login/LoginForm.tsx:16]: remove?

**Review by chownces — CHANGES_REQUESTED** [review]

File: frontend/src/sections/auth/login/LoginForm.tsx (hunk 2 above)
Comment (therizhao) [reply frontend/src/sections/auth/login/LoginForm.tsx]: done

File: frontend/src/sections/auth/login/LoginForm.tsx (hunk 1 above)
Comment (therizhao) [reply frontend/src/sections/auth/login/LoginForm.tsx]: done

**Comment by therizhao** [comment]
Removed unnecssary useffect

**Review by chownces — APPROVED** [review]

**Pull request merged**